<?xml version="1.0"?>
<launch>
    

    <include file="$(find eh_traj)/launch/executor.launch">
        <arg name="namespace" value="iris_0" />
    </include>

    <include file="$(find eh_traj)/launch/executor.launch">
        <arg name="namespace" value="iris_1" />
    </include>

    <include file="$(find eh_traj)/launch/dual_mission.launch" />

</launch>